prime-router/src/main/kotlin/azure/Event.kt

```diff
@@ -32,7 +32,8 @@ abstract class Event(val eventAction: EventAction, val at: OffsetDateTime?) {
         SEND_ERROR,
         WIPE_ERROR, // Deprecated
         RESEND,
-        REBATCH;
+        REBATCH,
+        FHIR_PROCESSING; // for HL7/FHIR conversions
```

Comment:
I am questioning the need for this event type. A message from a sender will either be full ELR or it will be a covid message, in which case it will only go into one pipeline to process (existing or FHIR). Why do we need a new event action - couldn't we just use `PROCESS` instead? We would know if it is FHIR or non-fhir based on sender.

Reply:
After conversing with Jim, Carlos, and Gage it is agreed we do not need this new Event type, and it will complicate downstream processes. This was needed when the idea was to split a submission into both pipelines; instead we will be using topics to have submissions go into one pipeline or the other.